src/scss/_typography.scss

```diff
@@ -0,0 +1,8 @@
+h2 {
+  margin-bottom: 1.8rem;
```

Comment:
Using such large margin-bottom values with no margin-top is going to cause issues with stacked headings + paragraphs (see attached)... we could add equivalent margin-top values, but Athena/BS4 generally avoids using margin-top anywhere to help avoid unwanted whitespace above headings within cards, jumbotrons, etc.  I'm not really sure what the best solution is for this besides letting the user adjust margins manually with utility classes.
![screen shot 2017-07-20 at 1 39 51 pm](https://user-images.githubusercontent.com/1649216/28433456-565f4ef4-6d51-11e7-9395-98a4520f24ad.png)